docs/user/02-configuration.md

```diff
@@ -73,15 +73,16 @@ Use the following sample CRs as guidance. Each can be applied immediately when y
 | **publisherService**                                 | string     |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
 | **specHash** (required)                              | integer    |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
-| **state** (required)                                 | string     | Can have one of the following values: `Ready`, `Error`, `Processing`, `Warning`. `Ready` state is set when all the resources are deployed successfully and backend is connected. It gets `Warning` state in case backend is not specified, NATS module is not installed or EventMesh secret is missing in the cluster. `Error` state is set when there is an error. `Processing` state is set if recources are being created or changed. See the next section for more details. |
+| **state** (required)                                 | string     | Can have one of the following values: `Ready`, `Error`, `Processing`, `Warning`. `Ready` state is set when all the resources are deployed successfully and backend is connected. It gets `Warning` state if backend is not specified, NATS module is not installed or EventMesh secret is missing in the cluster. `Error` state is set when there is an error not caused by a user. `Processing` state is set when eventing is initiliazed or backend is being switched. See the next section for more details. |
 
 <!-- TABLE-END -->
 
 <h3>More Details About Eventing CR State</h3>
```

Comment:
"more details" is very vague. The user's goal is probably to find out whether Eventing works or not, so the impact?
Even more useful would be that the users understand whether they must take action or not, but there's no column for that.

```suggestion
## Impact of the Eventing CR State
```

Reply:
It is actually "Impact of the Backend to the Eventing CR State and Event Flow". We document the backend impact.